components/phpunit_bridge.rst

```diff
@@ -293,6 +301,76 @@ Running the following command will display the full stack trace:
 
     $ SYMFONY_DEPRECATIONS_HELPER='/Doctrine\\Common\\ClassLoader is deprecated\./' ./vendor/bin/simple-phpunit
 
+Testing with multiple version of PHPUnit
+----------------------------------------
+
+Use Case
+~~~~~~~~
+
+When testing a library that have to be compatible with serveral version of PHP
+at the same time (like Symfony does), because of dependencies, the test suite
+have to be tested by differentes version of PHPUnit. Unfortunatly, writing a
+code compatible with a too wide range of version is not possible. ie:
+- several function in PHPUnit 8 are deprecated but the replacements methods
+  didn't exists int PHPUnit 4.
+- PHPUnit 8 added return typehint in method ``setUp(): void`` which is not
+  compatible with PHP 5.5.
+- PHPUnit switch to namespaced class starting from PHPUnit 6. Tests have to
+  handle both case with/without namespaces.
+
+Polyfill for the methods
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+When using the command ``simple-phpunit``, PHPUnit Bridge injects pollyfills
+for most of methods from the class ``TestCase`` and ``Assert`` in order to
```

Comment:
for most methods of the classes ``TestCase`` and ``Assert``. This allows writing test cases using the latest best practice while still remaining compatible with older PHPUnit versions.